### ISSUE TITLE
[Bug] Avg loss (and the rest) remains constant when training a VITS multi-speaker model in Catalan (Custom dataset) from scratch.

### ISSUE THREAD

**Issue by GerrySant** [coqui-ai/TTS#1931]
### Describe the bug

I am trying to train a multi-speaker VITS model in Catalan from scratch with 4 16GB V100 GPUs, however the loss of the model does not decrease. 
I have tried several things:
- Modifying hyperparameters like "lr_gen", "lr_disc", "batch_size", "eval_batch_size" among others.
- Assign "scheduler_after_epoch" to False.
- Train with only one GPU
- Training with only one speaker with 2h of speech.

### To Reproduce

Below I share the config, logs and a tensorboard capture of one of the trainings:
[config.txt](https://github.com/coqui-ai/TTS/files/9469027/config.txt)
[trainer_0_log.txt](https://github.com/coqui-ai/TTS/files/9469041/trainer_0_log.txt)
<img width="949" alt="Tensorboard" src="https://user-images.githubusercontent.com/90964413/187892752-a3b546c5-152f-4a90-9880-3f77362c5302.png">
<img width="940" alt="Tensorboard_2" src="https://user-images.githubusercontent.com/90964413/187893847-97897f45-344a-430b-be40-379f9b0c34c3.png">
<img width="1111" alt="Tensorboard3" src="https://user-images.githubusercontent.com/90964413/187903072-e06f3769-055b-4f3d-8acb-f69fa4b34eba.png">

About the datasets, I'm using 3 datasets for training:
- [festcat](http://festcat.talp.cat/devel.php), of which we have 13.36h of clean speech spread over 10 speakers.
- [OpenSLR](http://openslr.org/69/), of which we have 2.11h and 2.65h of clean speech split between male and female speakers respectively.
- CommonVoice, of which we have 102.5h of speech spread over approx. 200 speakers and with mostly acceptable quality.

### Environment

```shell
{
    "CUDA": {
        "GPU": [
            "Tesla V100-SXM2-16GB",
            "Tesla V100-SXM2-16GB",
            "Tesla V100-SXM2-16GB",
            "Tesla V100-SXM2-16GB"
        ],
        "available": true,
        "version": "10.2"
    },
    "Packages": {
        "PyTorch_debug": false,
        "PyTorch_version": "1.9.0a0+git3d70ab0",
        "TTS": "0.6.2",
        "numpy": "1.19.5"
    },
    "System": {
        "OS": "Linux",
        "architecture": [
            "64bit",
            "ELF"
        ],
        "processor": "ppc64le",
        "python": "3.7.4",
        "version": "#1 SMP Tue Sep 25 12:28:39 EDT 2018"
    }
}
```

### Additional context

Trainer was updated to `trainer==0.0.13`.

Please let me know if you need more information and thank you in advance.

**Comment by Edresson** [comment]
Hi @GerrySant , 

Could you verify if the training runs normally using just one GPU (CUDA_VISIBLE_DEVICES=0)? 

If my memory is not failing I got a similar problem some time ago. It was related to the mixed precision, could you try to change the "mixed_precision" to false in the training config?

**Comment by GerrySant** [comment]
Good morning @Edresson, 

This weekend I have launched two experiments with the changes you suggested. By setting `mixed_precision=False`, the loss decreases correctly. 

Thank you for the quick and accurate answer. 

I attach the images of the same training of this issue but now with `mixed_precision=False`.


<img width="941" alt="Tensorboard" src="https://user-images.githubusercontent.com/90964413/188378067-0f0e3c9b-4a06-497a-b4ab-d56d7284373b.png">
